core/runtime.rs

```diff
@@ -171,7 +155,7 @@ pub(crate) struct JsRuntimeState {
   dyn_module_evaluate_idle_counter: u32,
   pub(crate) js_error_create_fn: Rc<JsErrorCreateFn>,
   pub(crate) pending_ops: FuturesUnordered<PendingOpFuture>,
-  pub(crate) pending_unref_ops: FuturesUnordered<PendingOpFuture>,
+  pub(crate) unrefed_ops: HashSet<u64>,
```

Comment:
Should that be i64? You're casting `v8::Integer::value()` to u64 but that can give surprising results for negative numbers.

(Not really surprising, of course, just probably not what you want.)

Reply:
Sure, that makes more sense, maybe even `i32` as there are some other conversions going on in the code.